package-lock.json

```diff
@@ -4882,8 +4882,7 @@
       "version": "1.0.0",
       "resolved": "https://registry.npmjs.org/assert-plus/-/assert-plus-1.0.0.tgz",
       "integrity": "sha1-8S4PPF13sLHN2RRpQuTpbB5N1SU=",
-      "dev": true,
-      "optional": true
```

Comment:
@mrw100 were these changes to remove the optional property done by "npm install" or were they removed for another reason?

Reply:
These changes came from npm install